Python/ceval.c

```diff
@@ -990,7 +990,6 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
     #define DPRINTF(level, ...)
 #endif
 
-    OPT_STAT_INC(traces_executed);
     uint16_t uopcode;
```

Comment:
Heh, a declaration (`uint16_t uopcode;`) can't follow a label (`enter_tier_two:`), so you need a dummy statement (`;`) where this line was removed.

Reply:
Hm, the failure is sort of odd though. `_Py_JIT` should be defined, so this shouldn't exist at all. Weird.

Reply:
Huh.  Added.  I guess that's clang only?

Reply:
Oh, I see. The cross-compiled job builds twice. This is the first (non-JIT) run failing.

And yeah, seems like only Clang cares.